### ISSUE TITLE
MixinExtras conflict with Debugify on server

### ISSUE THREAD

**Issue by FlorianFreudiger** [Faboslav/friends-and-foes#78]
### Minecraft version information

1.19.2

### Mod loader information

Fabric

### Mod loader version information

0.14.10 (Fabric)

### Mod version information

1.6.1

### Expected Behavior

Successful server start

### Actual Behavior

Server fails starting
```[01:41:46] [main/ERROR]: A mod crashed on startup!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'debugify'!
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:164) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) [fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.14.10.jar:?]
	at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:69) [fabric-server-mc.1.19.2-loader.0.14.10-launcher.0.11.1.jar:0.11.1]
Caused by: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'debugify'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:162) ~[fabric-loader-0.14.10.jar:?]
	... 4 more
Caused by: java.lang.LinkageError: loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @305b7c14 attempted duplicate class definition for com.llamalad7.mixinextras.MixinExtrasBootstrap. (com.llamalad7.mixinextras.MixinExtrasBootstrap is in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @305b7c14, parent loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader$DynamicURLClassLoader @c0c2f8d)
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:186) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:347) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145) ~[fabric-loader-0.14.10.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.fabricmc.loader.impl.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:117) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:53) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:162) ~[fabric-loader-0.14.10.jar:?]
	... 4 more
2022-11-29T01:41:46.814+0100	WARN	mc-server-runner	Minecraft server process failed	{"exitCode": 1}
2022-11-29T01:41:46.815+0100	INFO	mc-server-runner	Done
```

### Reproduction Steps

1. Setup fabric server
2. Add both Debugify and Friends&Foes
3. Start server

**Comment by FlorianFreudiger** [comment]
Searching further I discovered https://github.com/isXander/Debugify/issues/173 which explains that this conflict is caused by MixinExtras being initialized manually
https://github.com/Faboslav/friends-and-foes/blob/81974f175cd89beda645980223d4309e1f568f4c/common/src/main/java/com/faboslav/friendsandfoes/util/plugin/MixinExtrasPlugin.java#L14

instead of in the "preLaunch" step of the fabric.mod.json like added in this commit https://github.com/isXander/Debugify/commit/12182ad442d291f6adc428380eec7d3bc3e78a35

**Comment by FlorianFreudiger** [comment]
#69 is probably related to this issue

**Comment by Faboslav** [comment]
Hello thank you for the detailed report very much, i will try to make a fix in the next release 1.6.2.

**Comment by Faboslav** [comment]
Should be resolved with https://github.com/Faboslav/friends-and-foes/releases/tag/mc1.19.2-1.6.2.
